pkg/endpoint/bpf.go

```diff
@@ -499,6 +499,13 @@ func (e *Endpoint) regenerateBPF(owner Owner, epdir, reason string) (uint64, err
 			e.Mutex.Unlock()
 			return 0, err
 		}
+		// Clean up map contents
+		log.Debugf("Flushing old policies map")
```

Comment:
Doing this here means we would have some time where the map was empty, blocking all traffic, before we repopulate it at the end of the function, right?

Reply:
>blocking all traffic

depends on the endpoint's configuration, if ingress policy is enabled / disabled